app/api/interfaces/interfaces.tsx

```diff
@@ -0,0 +1,18 @@
+export interface TaskSheet {
+    id: number,
+    tasks: string ///Task[]; //in SQL database this would be a json
```

Comment:
I would add a description field to the TaskSheets. As noted in the Project document, we want the TaskSheets to hold the "owner(s)" and "assignee(s)", maybe as a string (json list)?

Reply:
This is a list of Task.

Reply:
We don't need the owner and assignee. That is saved in the User table.

Reply:
The description field would be useful.

Reply:
@veryyynice